docs/investigation/2022-06-20-cfg-via-crds.md

```diff
@@ -0,0 +1,581 @@
+# Configure BotKube via CRs
```

Comment:
General comment to all these investigation documents:
It would be great to unify terms (and also describe them, as in the proposal). If that's too much work, let's have the proposal from #626 having agreed terms (e.g. `notifier` will become something different, as long as you agree with me) and here let's just describe used terms for each document (the `notifier` could stay as it is).